### PR TITLE
Feat: Added cart page to my e-com site.

### PR DESCRIPTION
**Pull request by zoef-yt**
Added cart page to my e-com site.
> image

### CODE REVIEW

**Review by j24m — APPROVED** [review]
Looks good to merge !!

File: screens/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	width: 100%;
+}
+
+.suggestions-section > * {
+	margin: 2rem;
+	padding: 1rem;
+}
+
+.suggestions-row {
+	max-width: 100%;
```
Comment (amartya-rs) [inline screens/cart/cart.css]: width: 100% is enough I guess.

File: screens/cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+	align-items: center;
+	justify-content: center;
+	gap: 2rem;
+}
+
+.cartpage-product-img {
+	max-width: 20rem;
+	object-fit: contain;
+	background-color: var(--colour-perfect-white);
+	aspect-ratio: 1/1;
```
Comment (itsmebipul00) [inline screens/cart/cart.css]: or just 1

**Review by itsmebipul00 — APPROVED** [review]

**Comment by itsmebipul00** [comment]
Nice work

**Review by varun13169 — APPROVED** [review]
LGTM, nice work.

File: screens/cart/cart.css (hunk 2 above)
Comment (zoef-yt) [reply screens/cart/cart.css]: cool will change it

File: screens/cart/cart.css (hunk 1 above)
Comment (zoef-yt) [reply screens/cart/cart.css]: will change it

**Review by amartya-rs — APPROVED** [review]

**Pull request merged**